README.md

```diff
@@ -104,7 +104,28 @@ $ vault auth enable -plugin-name='jwt' plugin
 Successfully enabled 'plugin' at 'jwt'!
 ```
 
-#### Tests
+### Provider-specific configuration
```

Comment:
```suggestion
### Provider-specific handling
```

I think we should differentiate between provider specific configuration (config map that users input in HCL) and handling (code written by developers). What do you think?

Reply:
Sure, I like that distinction.